DB/optionsDB.lua

```diff
@@ -40,6 +40,8 @@ function CraftSim.DB.OPTIONS:Migrate()
                 .tsmPriceKeyMaterials
             CraftSimDB.optionsDB.data[CraftSim.CONST.GENERAL_OPTIONS.TSM_PRICE_KEY_ITEMS] = CraftSimOptions
                 .tsmPriceKeyItems
+            CraftSimDB.optionsDB.data[CraftSim.CONST.GENERAL_OPTIONS.TSM_RESTOCK_KEY_ITEMS] = CraftSimOptions

```

Comment:
This does not make sense ^^
the migration here is for cases when users switch from a version that has the old options global to the new optionsDB so it transforms all old values into the new db

since you added your option to the defaults adding a migration for it should not be necessary